oauth/v1/types.go

```diff
@@ -117,6 +121,16 @@ type OAuthClient struct {
 	// AccessTokenMaxAgeSeconds overrides the default access token max age for tokens granted to this client.
 	// 0 means no expiration.
 	AccessTokenMaxAgeSeconds *int32 `json:"accessTokenMaxAgeSeconds,omitempty" protobuf:"varint,8,opt,name=accessTokenMaxAgeSeconds"`
+
+	// AccessTokenInactivityTimeoutSeconds overrides the default token timeout for tokens granted to this client.
+	// This option is meaningful only if the feature is enabled by setting the
+	// corresponding configuration option in the master configuration.
+	// This value needs to be set only if the default set in configuration is
+	// not appropriate for this client. Valid values are:
```

Comment:
You need to define "inactivity", i.e. what this is.  Something like:

`This timeout sets an upper bound on how frequently the token must be used.  If a token is not used for longer than the inactivity timeout it will be deleted.  This allows the token to have a longer expiration period but only if used continuously`

(wording needs some work).  Basically someone needs to know what "times out" means here and in the OAuthAccessToken.

Reply:
Ok if we need a treaty I'll write a treaty.
My impression was we want to keep this comments relatively small.
Note, the token is not necessarily deleted, but it will fail authentication after the timeout, so I am not explicitly saying it is deleted as that is an implementation detail.